### PR TITLE
chore: add CODEOWNERS

### PR DESCRIPTION
**Pull request by LayZeeDK**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[x] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?
No pre-assigned reviewers.

Issue Number: N/A

## What is the new behavior?
- The @ngworker/ngworkers team is the default code owner.
- @LayZeeDK is the code owner for `packages/spectacular/**`.

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

## Other information
See [About code owners](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners).

### CODE REVIEW

**Review by NachoVazquez — COMMENTED** [review]
It looks good, but I'm not sure how it works.

Additionally, should you update the owner name at the spectacular Readme?

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# The folders are ordered as follows:
+
+# In each subsection folders are ordered first by depth, then alphabetically.
+# This should make it easy to add new rules without breaking existing ones.
+
+# Global rule:
+*           @ngworker/ngworkers
+
+# Spectacular:
+/packages/spectacular/**  @LayZeeDK
```
Comment (NachoVazquez) [inline .github/CODEOWNERS:18]: I think the right way would be

`/packages/spectacular/ @LayZeeDK ` 

See:

```
# The `docs/*` pattern will match files like
# `docs/getting-started.md` but not further nested files like
# `docs/build-app/troubleshooting.md`.
docs/*  docs@example.com

# In this example, @octocat owns any file in an apps directory
# anywhere in your repository.
apps/ @octocat

# In this example, @doctocat owns any file in the `/docs`
# directory in the root of your repository and any of its
# subdirectories.
/docs/ @doctocat
```

**Review by NachoVazquez — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (LayZeeDK) [reply .github/CODEOWNERS:18]: I got it from https://github.com/dotnet/samples/blob/master/.github/CODEOWNERS#L18

Comment (NachoVazquez) [reply .github/CODEOWNERS:18]: I got it from here https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners

Comment (NachoVazquez) [reply .github/CODEOWNERS:18]: I think they may be the same.

Comment (LayZeeDK) [reply .github/CODEOWNERS:18]: `(...)/**` makes more sense to me :-)

Comment (NachoVazquez) [reply .github/CODEOWNERS:18]: Me too, but I got confused while reading the docs.

Comment (NachoVazquez) [reply .github/CODEOWNERS:18]: Would you add your name to the spectacular Readme and/or package.json?

**Review by santoshyadavdev — APPROVED** [review]
LGTM

File: .github/CODEOWNERS (hunk 1 above)
Comment (LayZeeDK) [reply .github/CODEOWNERS:18]: Maybe package.json. A code owner icon should be displayed in GitHub when browsing that part of the workspace.

**Review by SerkanSipahi — APPROVED** [review]
LGTM

**Review by NachoVazquez — APPROVED** [review]

**Pull request merged**